### DISCUSSION TITLE
Can´t connect with stun server.

### DISCUSSION THREAD

**Discussion by eyanson** [miroslavpejic85/p2p#50] [Announcements]
I receving this message:
"Your external IP can´t be obtained. Could not find a working STUN server:-("

Can you help me?

**Comment by miroslavpejic85** [comment]
Hi @eyanson, 
Check out: 👉 https://github.com/miroslavpejic85/p2p/issues/45#issuecomment-1339422032 👈

**Reply by eyanson** [reply]
Hi @miroslavpejic85 . Can you help me with this error?
Thanks in advance
